topics/ecology/tutorials/QGIS_Web_Feature_Services/tutorial.md

```diff
@@ -0,0 +1,136 @@
+>    > ![Image of QGIS interface to show you to click on the button "effect drop shadow" to make the south pole more visible on your map.](../../images/QGIS/qgistuto5.PNG)
+>    > ![Image of QGIS interface to show you how to add a title and change the color of your south pole marker](../../images/QGIS/qgistuto6.PNG)
+> 12. And to "antarctica_country_border".
+>    > ![Image of QGIS interface showing how to change the background or color of the geographical zone here Antarctica](../../images/QGIS/qgistuto7.PNG)
+> 
+{: .hands_on}
+
+## Querying a WFS layer
+
+Even if you can select, download and display entire WFS layers, it is often more efficient to interrogate a layer before load it to QGIS. This is a major interest of the use of such web services, as you can save internet bandwith selecting only part of layers you want in your own QGIS instance.
+
+> <hands-on-title>Querying a WFS layer</hands-on-title>
+>
+> 1. In the WFS server already created in the first step of this tutorial, there is layer called `countries (excluding Antarctica)`. If we want to know where is South Africa related to the `south_poles_wfs`, there is several manner to operate. One can load the entire layer `countries (excluding Antarctica)` and then using it locally, or we can save bandwith and only load locally the needed informations concerning South Africa. We will here use the second manner, querying the WFS layer to obtain only information we will use in our QGIS instance.
+> 2. Go to the top left of QGIS and click on "Layer"
+> 3. Click on the 1st option "Data Source Manager" 
+> 4. Select "WFS / OGC API - Features" then the already connected `nsidc` server
+> 5. Select the `countries (excluding Antarctica)` layer and click "Build query" button
+>    > ![Image of QGIS interface to help you see which layers to select in the WFS / OGC API features](../../images/QGIS/qgistuto8.PNG)
+> 6. On the new dialog box, you can copy/paste this query: `SELECT * FROM country_borders_excluding_antarctica WHERE "Countryeng" = 'South Africa'` using the clipboard functionnality from the left QGIS panel to make a "bridge" between your local clipboard and the virtualized QGIS one.
+>    > ![Image of QGIS interface to help you see what to write in the "SQL query composer"](../../images/QGIS/qgistuto9.PNG)
+> 7. Clicking ok, you can now see the SQL query on a dedicated column of the layers table.
+>    > ![Image of QGIS interface showing you the results of the SQL command on the selection of layers in the WFS / OGC API features](../../images/QGIS/qgistuto10.PNG)
+> 9. Clicking the "Add" button, you now have the South Africa layer added and diplayed on QGIS.
+>    > ![Image of QGIS interface showing the resulting map of all your changes since the beginning of the tutorial](../../images/QGIS/qgistuto11.PNG)
```

Comment:
```suggestion
>    ![Image of QGIS interface showing the resulting map of all your changes since the beginning of the tutorial](../../images/QGIS/qgistuto11.PNG)
```

please remove the `>` that are in front of the images, they mark the beginning of a block quote and are unnecessary.

Reply:
(This applies to all of the imgaes it seems, if you're in the "Files" https://github.com/galaxyproject/training-material/pull/4381/files tab you'll see the linting error messages about a potentially broken box.